tools/assetpipelineeditor/orbitcameracontroller.cpp

```diff
@@ -294,10 +294,12 @@ void OrbitCameraController::setCamera(Qt3DRender::QCamera *camera)
 
         m_camera = camera;
 
-        if (m_camera)
-            connect(m_camera, &Qt3DCore::QNode::nodeDestroyed, this, [this]() {
-                setCamera(nullptr);
+        if (m_camera) {
+            connect(m_camera, &Qt3DCore::QNode::nodeDestroyed, this, [this, camera]() {
+                if (camera == this->m_camera)
```

Comment:
I dont get why the if

Reply:
hard to see in the PR but this a separate commit with a message explaining.

When the camera is loaded from a gltf file, loading a second file will
destroy the first camera using deleteLater. By that time a new camera
will have been selected and we want to keep using that.